streaming/src/test/org/apache/hive/streaming/TestStreaming.java

```diff
@@ -964,8 +964,8 @@ private void checkDataWritten(Path partitionPath, long minTxn, long maxTxn, int
         min = pd.getMinWriteId();
       }
     }
-    Assert.assertEquals(minTxn, min);
-    Assert.assertEquals(maxTxn, max);
+    Assert.assertEquals(minTxn + 1, min);
```

Comment:
I think it will be helpful to add a comment on why there is a +1 since I not trivial to understand the test case otherwise.

Reply:
Added a comment